### ISSUE TITLE
Vertical scroll is not working properly in Firefox and IE 9, IE Edge when scroll bar dragged with mouse

### ISSUE THREAD

**Issue by nithinbandaru1** [almende/vis#3353]
Hi,

I have been working with vis timeline form past few weeks.

I consider **infinity scrolling with vertical scroll(visible scroll bar)** is one of its best feature.

But there is compatibility issues while working with **Firefox and IE 9, IE Edge**.

I added a jsbin link to look at the issue I am facing.

http://jsbin.com/caxogomufi/1/edit?html,js,output

In Firefox the scrollbar is **very very slow while dragging it with mouse**. But with mouse drag is fast.

In IE and Edge the scrollbar works fast except ** once i release the mouse after dragging it is not released and messing up**.

Is there any additional CSS or JS that I need to add to make them compatible and work smoothly?

**Comment by nithinbandaru1** [comment]
Hi guys,
Is any one out there facing this kind of issue. Please reply.
I at-least expect it to work with firebox correctly. But it is very frustrating with the slow scroll issue.

**Comment by SpeedHighway** [comment]
Yes, I have the same issue with Firefox 57 and Firefox for Developers 58b.  Vertical scrolling is effectively unusable in Firefox, but in Edge, where everything else breaks, it works flawlessly.

**Comment by SpeedHighway** [comment]
Could this be a similar issue to this?

https://github.com/harvesthq/chosen/issues/2823

The jsfiddle that they gave does the same thing on the list, in that dropdown, on Firefox, as what happens with the vis.js timeline.

**Comment by guvial** [comment]
Hi,
dragging the scroll bar and scrolling with the mouse wheel do not work on my FireFox 52.7.2 (windows). Do you already have an idea about the root cause? I could also help to fix this issue.

**Comment by cm0s** [comment]
The bug is still occuring on the latest version of vis 4.21.0.
